tias/src/Components/common.scss

```diff
@@ -0,0 +1,303 @@
+@use "sass:color";
+
+$green : #79ca42;
+$blue : #33ACE3;
+$accent : #800000;
+$render_background : #d3dbdc;
+$render_divider : #85929C;
+
+.App {
+	display: flex;
+	flex-direction: row;
+
+	height: 100vh;
+	width: 100vw;
+}
+
+.slim {
+	padding: 0 !important;
+	margin: 0  !important;
+	padding: 0 !important;
+}
+
+.vstack.main {
+	flex-grow: 1;
+	border-left: 2px solid;
+	border-right: 2px solid;
+
+	padding-left: 15px;
+	padding-right: 15px;
+}
+
+.hr-container {
+	padding-left: 10px;
+	padding-right: 10px;
+}
+
+.scrollable::-webkit-scrollbar {
+	display: none;
+}
+
+.scrollable {
+	overflow-y: scroll;
+	overflow-x: hidden;
+	flex: 1;
+
+	// Hide the scrollbar
+	-ms-overflow-style: none;  /* IE and Edge */
+	scrollbar-width: none;  /* Firefox */
+}
+
+.vstack {
+	display: flex;	
+    flex-direction: column;
+
+	min-width: 20px;
+
+
+	.header {
+		margin: 0;
+		border: 0;
+		padding: 0;
+		padding-left: 5px;
+		padding-right: 5px;
+
+		border-bottom: 2px solid;
+		margin-bottom: 5px;
+	}
+
+	.hstack.filter {
+		border: 2px solid;
+		margin-top: 10px;
+		margin-bottom: 10px;
+		padding: 0;
+
+		background-color: $accent;
+	}
+
+	.filter.element {
+		color: white;
+		line-height: 20px;
```

Comment:
TODO: Refactor units to relative units: https://www.w3schools.com/CSSref/css_units.asp